runhouse/resources/serverless/aws_lambda_function.py

```diff
@@ -58,24 +55,33 @@ class AWSLambdaFunction(Function):
     ]
     GEN_ERROR = "could not create or update the AWS Lambda."
     FAIL_CODE = 1
-    DEFAULT_REGION = "us-east-1"
-    DEFAULT_PY_VERSION = "python3.9"
     EMPTY_ZIP = -1
+    MAX_WAIT_TIME = 60  # seconds, max time that can pass before we raise an exception that AWS update takes too long.
+    DEFAULT_REGION = "us-east-1"
+    DEFAULT_RETENTION = 30  # one month, for lambdas log groups.
+    DEFAULT_TIMEOUT = 30  # seconds
+    DEFAULT_MEMORY_SIZE = 128  # MB
+
+    if Path(CRED_PATH).is_file():
```

Comment:
do these still need to be class attributes? we could also initialize them as part of the constructor